SpatialGDK/Source/SpatialGDK/Private/SpatialView/AuthorityRecord.cpp

```diff
@@ -0,0 +1,57 @@
+#include "SpatialView/AuthorityRecord.h"
+
+namespace SpatialGDK
+{
+
+void AuthorityRecord::SetAuthority(Worker_EntityId EntityId, Worker_ComponentId ComponentId, Worker_Authority Authority)
+{
+	const EntityComponentId Id = {EntityId, ComponentId};
```

Comment:
My brain broke a little reading this function (RemoveSingle == 0 and RemoveSingle >1 tricked me). Would appreciate some guiding comments.

Reply:
Will add.

Reply:
This is one of the cases I think the == explicit value makes it harder to read.
`if (!RemoveSingle)` seems nicer.
Comments should be added to this either way, but it might be a good time to break from the convention.

Reply:
I agree - will leave it to your discretion